### DISCUSSION TITLE
sample_by_individuals() & rewriting p4_ to p5_

### DISCUSSION THREAD

**Discussion by maja-nagler** [IBAHCM/RPiR#556] [Q&A for project]
How is the function sample_by_individuals() supposed to be different from an already existing sample() function? Are we expected to generate a new datasets necessarily with the use of  rmultinom()?

https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/sample

Do You expect us to rewrite p4_  functions to adapt them to the 5th part (p5_ form)?

**Comment by richardreeve** [comment]
@maja-nagler - it's not clear to me what differences there would be between a practical 4 version of any of the functions vs a practical 5 version. If you don't see any difference, then it's always best to reuse an existing function rather than re-writing one.

As fas as `sample()` vs `sample_by_individuals()`, these function are different though. `sample()` picks some of the elements of a vector:

```R
> sample(1:10, size = 5)
[1] 3 9 5 7 4
```

vs `sample_by_individuals()` which picks some of the individuals in a vector:

```R
> sample_by_individuals(1:10, count = 5)
[1] 0 1 0 0 0 1 0 0 2 1
```
